webapp/src/routes/app/+page.svelte

```diff
@@ -1,53 +1,221 @@
 <script lang="ts">
+    import { onMount, onDestroy } from 'svelte';
     import Navbar from '$lib/components/navbar.svelte';
-    import { MapLibre } from 'svelte-maplibre';
+    import { MapLibre, DefaultMarker, Popup, GeolocateControl } from 'svelte-maplibre';
+    import SpotsListComponent from '$lib/components/spot-listings/spots-list-component.svelte';
+    import { spots_data } from './your-spots/mock_data';
+    import { Search } from 'carbon-components-svelte';
+
+    const apiKey = import.meta.env.VITE_API_KEY;
+
+    let mapCenter: [number, number] = [-97.1, 49.9];
```

Comment:
It is hardcoded to Winnipeg, the center requires a number array of length to know what the default location of the map upon load would be. It can't be the location of the booker because for that the user needs to give permission for their location access (which they can do once on the map using the geolocate control on the bottom left).

Reply:
If that is the case, I think we should at least have a constant for it somewhere.